frontend/src/modules/editor/components/TranslationLength.js

```diff
@@ -0,0 +1,87 @@
+/* @flow */
+
+import * as React from 'react';
+
+import './TranslationLength.css';
+
+import type { DbEntity } from 'modules/entitieslist';
+
+
+type Props = {|
+    entity: ?DbEntity,
+    translation: string,
+|};
+
+
+/*
+ * Shows translation length vs. original string length, or countdown.
+ * 
+ * Countdown is current only supported for LANG strings, which use special
+ * syntax in the comment to define maximum translation length. MAX_LENGTH
+ * is provided for strings without HTML tags, so they need to be stripped.
+ */
+export default class TranslationLengthBase extends React.Component<Props> {
+    getLimit() {
+        const entity = this.props.entity;
+
+        if (!entity) {
+            return null;
+        }
+
+        const split = entity.comment.split('\n');
```

Comment:
I like the name `parts` better because it's not the same as the function, avoiding confusion when reading this kind of code.

Reply:
Also makes refactoring easier. :-)